### ISSUE TITLE
result of AN4

### ISSUE THREAD

**Issue by seed93** [SeanNaren/CTCNetworkClient#15]
I am new to speech and torch. I just want to check whether I run your example correctly.
It took about 28 mins using 1 GTX 970 GPU and got a model whose size is 940M.
WER is 89.13 and with content based  spell check is 83.31.
Are these all right? Hope to get any help.
Thank you for your wonderful work.

**Comment by TangXing** [comment]
WER is 89.13%?

**Comment by seed93** [comment]
yes, I guess it is wrong

**Comment by SeanNaren** [comment]
hey could you try on the latest branch of master? I'm having difficulties my end on my ubuntu installation I need to fix ASAP.

**Comment by seed93** [comment]
I tested with commit a180e4b144402179013cf583b51c566521f2c866

**Comment by SeanNaren** [comment]
I've done some modification to remove that batchnorm, it doesn't seem to be correct so I've reverted it.

**Comment by seed93** [comment]
Now the WER is 86.03 and 80.85. The final loss is 6.75, maybe training epoch is not large enough?

**Comment by SeanNaren** [comment]
Right I've fixed up my ubuntu installation, I'll verify on my end to find the problem.

**Comment by SeanNaren** [comment]
Could you try the latest master branch? cheers!

**Comment by seed93** [comment]
Still not right. Get loss 83.02 after 70 epochs.

**Comment by SeanNaren** [comment]
I'm not entirely sure what this issue is (it runs fine on my side), could you pull the branch 'ModifiedCriterion' and run the AN4 training script for me?

**Comment by seed93** [comment]
Still not right.
I download AN4 of "Raw audio (.raw) format, little endian byte order ", is that right? Then use your .sh tu get proper format to run.

**Comment by SeanNaren** [comment]
Ah this might be the issue, try downloading the big endian byte order one, I'll update the readme to include a description of how to download/extract it!

Edited the bottom of the [Install](https://github.com/SeanNaren/CTCSpeechRecognition/blob/master/INSTALL.md) with Linux commands to download/extract.

**Comment by seed93** [comment]
That's the point. It looks much better. Thanks!

**Comment by SeanNaren** [comment]
Awesome cheers!
